.gitignore

```diff
@@ -1,52 +1,143 @@
-# Temporary Python files
-*.pyc
-*.egg-info
+.drafts
+config.json
 __pycache__
-.ipynb_checkpoints
-setup.py
+
+.vscode
+.secrets.json
+
+# CUSTOM FILES!
+
+images/
+playground/
+
+# CONFIG FILES
+mensa_config.py
+app/api/data/
+app/api/dummydata/speiseplan
+
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+.idea/
+# C extensions
+*.so
+.DS_Store
+# Distribution / packaging
+.Python
+build/
+develop-eggs/
+dist/
+000*.json
+downloads/
+eggs/
+.eggs/
+lib/
+lib64/
+parts/
+sdist/
+var/
+wheels/
 pip-wheel-metadata/
+share/python-wheels/
+*.egg-info/
+.installed.cfg
+*.egg
+MANIFEST
+
+# PyInstaller
+#  Usually these files are written by a python script from a template
+#  before PyInstaller builds the exe, so as to inject date/other infos into it.
+*.manifest
+*.spec
+
+# Installer logs
+pip-log.txt
+pip-delete-this-directory.txt
+
+# Unit test / coverage reports
+htmlcov/
+.tox/
+.nox/
+.coverage
+.coverage.*
+.cache
+nosetests.xml
+coverage.xml
+*.cover
+*.py,cover
+.hypothesis/
+.pytest_cache/
+
+# Translations
+*.mo
+*.pot
+
+# Django stuff:
+*.log
+local_settings.py
+db.sqlite3
+db.sqlite3-journal
 
-# Temporary OS files
-Icon*
+# Flask stuff:
+instance/
+.webassets-cache
 
-# Temporary virtual environment files
-/.cache/
-/.venv/
+# Scrapy stuff:
+.scrapy
 
-# Temporary server files
+# Sphinx documentation
+docs/_build/
+
+# PyBuilder
+target/
+
+# Jupyter Notebook
+.ipynb_checkpoints
+
+# IPython
+profile_default/
+ipython_config.py
+
+# pyenv
+.python-version
+
+# pipenv
+#   According to pypa/pipenv#598, it is recommended to include Pipfile.lock in version control.
+#   However, in case of collaboration, if having platform-specific dependencies or dependencies
+#   having no cross-platform support, pipenv may install dependencies that don't work, or not
+#   install all needed dependencies.
+#Pipfile.lock
+
+# celery beat schedule file
+celerybeat-schedule
+
+# SageMath parsed files
+*.sage.py
+
+# Environments
 .env
-*.pid
-
-# Generated documentation
-/docs/gen/
-/docs/apidocs/
-/site/
-/*.html
-/docs/*.png
-
-# Google Drive
-*.gdoc
-*.gsheet
-*.gslides
-*.gdraw
-
-# Testing and coverage results
-/.coverage
-/.coverage.*
-/htmlcov/
-
-# Build and release directories
-/build/
-/dist/
-*.spec
+.venv
+env/
+venv/
+ENV/
+env.bak/
+venv.bak/
+
+# Spyder project settings
+.spyderproject
+.spyproject
+
+# Rope project settings
+.ropeproject
 
-# Sublime Text
-*.sublime-workspace
+# mkdocs documentation
+/site
 
-# Eclipse
-.settings
+# mypy
+.mypy_cache/
+.dmypy.json
+dmypy.json
 
-# Custom
-Session.vim
-pytest.ini
-.pylint.ini
+# Pyre type checker
+.pyre/
```

Comment:
So most of the additions don't make a whole lot of sense for the project, this project doesn't work with a lot of the tools. I agree with the changes that remove unnecessary entries like the sublime, google drive entries. But most of your additions look like they come from your personal gitignore, which shouldn't be added to a general purpose gitignore file but should be part of your local file.

Reply:
Nope, it's [GitHubs own Python gitignore Boilerplate](https://github.com/github/gitignore/blob/main/Python.gitignore). 

But as said, somehow a few (4) of my personal ignores ended up in this commit. 
I double-checked and clean them out. 
 👍